go-base/Makefile

```diff
@@ -39,18 +39,19 @@ gobootstrap: FORCE
 	curl -L https://storage.googleapis.com/golang/go1.6.2.linux-amd64.tar.gz | tar xz -C $(GOBOOTSTRAP) --strip-components=1 
 
 .PHONY: gorump
-gorump: gobootstrap rumprun solo5 FORCE
-	source $(RUMPRUN)/obj/config-PATH.sh && cd $(GORUMP)/go/src && CGO_ENABLED=0 GOROOT_BOOTSTRAP=$(GOBOOTSTRAP) GOOS=rumprun GOARCH=amd64 ./make.bash && cd $(GOBASE)
+gorump: gobootstrap rumprun FORCE
+	source $(RUMPRUN)/obj/config && source $(RUMPRUN)/obj/config-PATH.sh && \
+		cd $(GORUMP)/go/src && CGO_ENABLED=0 GOROOT_BOOTSTRAP=$(GOBOOTSTRAP) GOOS=rumprun GOARCH=amd64 ./make.bash && cd $(GOBASE)
```

Comment:
Not really a problem for this PR, but in future for aarch64, we should make `GOARCH` a variable. We can open an issue for this?

Reply:
Ahh, didn't notice that. Was grepping for x86, will look for amd64 as well. Thanks!

Reply:
Creating an issue to discuss the proper way of having multiple archs